cli/EAP7-924_Coloured_output_for_CLI.adoc

```diff
@@ -0,0 +1,83 @@
+= EAP7-924 Coloured output for CLI
+:author:            Ingo Weiss
+:email:             ingo@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+The intent of this feature is to provide a better user experience by means of colouring important parts of the CLI workflow, this includes error messages, warning messages, successful outputs and better visual feedback when using batch and flow control mechanisms if/for/while.
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}EAP7-924[EAP7-924]
+
+=== Related Issues:
+
+* {issue-base-url}WFCORE-3450
+* {issue-base-url}WFCORE-3577
+
+=== Dev Contacts:
+
+* mailto:ingo@redhat.com[Ingo Weiss]
+* mailto:jdenise@redhat.com[Jean-François Denise]
+
+=== QE Contacts:
+
+* TBD
+
+=== Affected Projects or Components:
+
+* WildFly Core CLI
+
+
+== Requirements
+
+=== Hard requirements
+
+* Ability to enable or disable colour output via CLI argument
```

Comment:
What is the default value? Should the colours be enabled by default? (cc: @jfdenise, @bmaxwell ) The answer needs to be mentioned in this proposal.

Reply:
It should be enabled by default unless the CLI is invoked in a non-interactive way, for example, when running scripts, like the examples below.